phase1/master.cfg

```diff
@@ -562,7 +562,7 @@ c["schedulers"].append(
 
 
 def IsNoMasterBuild(step):
-    return step.getProperty("branch") != "main"
+    return step.getProperty("branch") != "main" and step.getProperty("branch") != "master"
```

Comment:
```suggestion
    branch = step.getProperty("branch")
    return branch not in ['main', 'master']
```

nitpick, looks more idiomatic/readable?

Reply:
Sure please go ahead

Reply:
You're signing commits, feel free to merge it yourself so the signature is kept.